### PR TITLE
Add API for Events and Rebase branch

### PR DESCRIPTION
**Pull request by jackxujh**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,3 +5,7 @@ django-heroku==0.3.*
 django-crispy-forms==1.8.*
 python-dotenv==0.10.*
 whitenoise==4.1.4
+djangorestframework
```
Comment (VentusXu09) [inline requirements.txt:9]: We will use DRF on our project?

Comment (jackxujh) [reply requirements.txt:9]: Yes, it makes things easier.

Comment (jackxujh) [reply requirements.txt:9]: Like, by a ton easier.

Comment (VentusXu09) [reply requirements.txt:9]: All right, please introduce DRF to the team. 
Do we need to make changes to old APIs like logout? If so, we need to add an issue to the next Sprint.

**Review by Ricardo-1894 — APPROVED** [review]
LGTM

**Comment by alldne** [comment]
It seems this pr contains changes from the previous repository. How about creating another PR and merging it first just for the purpose of rebase since it's gonna be trivial changes that we didn't write.

After that, we can rebase this PR with a new `develop` as base.

File: mercury/urls.py [hunk 2]
```diff
@@ -11,4 +29,18 @@
     path("stopwatch/", stopwatch.StopwatchView.as_view(), name="stopwatch"),
     path("api/can/", can.post, name="can-api"),  # CAN API Ingestion endpoint
     path("can/", can.CANUI.as_view(), name="can-ui"),  # CAN Decoder UI endpoint
+    path("api/v1/", include("api.urls"), name="api"),
+    # re_path(
+    #     r"^swagger(?P<format>\.json|\.yaml)$",
+    #     schema_view.without_ui(cache_timeout=0),
+    #     name="schema-json",
+    # ),
```
Comment (alldne) [inline mercury/urls.py:38]: Better delete any unused & commented lines.

File: mercury/templates/style.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
...b9cc}.custom-range:disabled::-moz-range-track{cursor:default}.custom-range:disabled::-ms-thumb{background-color:#b7b9cc}.custom-control-label::before,.custom-file-label,.custom-select{transition:background-color .15s ease-in-out,border-color .15s ease-in-out,box-shadow .15s ease-in-out}@media (prefers-reduced-motion:reduce){.custom-control-label::before,.custom-file-label,.custom-select{transition:none}}.nav{display:flex;flex-wrap:wrap;padding-left:0;margin-bottom:0;list-style:none}.nav-link{display:block;padding:.5rem 1rem}.nav-link:focus,.nav-link:hover{text-decoration:none}.nav-link.disabled{color:#858796;pointer-events:none;cursor:default}.nav-tabs{border-bottom:1px solid #dddfeb}.nav-tabs .nav-item{margin-bottom:-1px}.nav-tabs .nav-link{border:1px solid transparent;border-top-left-radius:.35rem;border-top-right-radius:.35rem}.nav-tabs .nav-link:focus,.nav-tabs .nav-link:hover{border-color:#eaecf4 #eaecf4 #dddfeb}.nav-tabs .nav-link.disabled{color:#858796;background-color:transparent;border-color:transparent}.nav-tabs .nav-item.show .nav-link,.nav-tabs .nav-link.active{color:#6e707e;background-color:#fff;border-color:#dddfeb #dddfeb #fff}.nav-tabs .dropdown-menu{margin-top:-1px
```
Comment (alldne) [inline mercury/templates/style.css:10]: Is this changes from upstream?

**Comment by alldne** [comment]
I rebased it against our current develop

**Pull request closed without merging**